src/main/java/com/vention/stock_market_share/repository/UserRepository.java

```diff
@@ -61,9 +59,10 @@ public User findById(Long id) {
     public long save(User user ) {
         try (Connection connection = dataSource.getConnection();
              PreparedStatement preparedStatement = connection.prepareStatement(SQL_INSERT, Statement.RETURN_GENERATED_KEYS)) {
+            if (isValidInput(user)){
```

Comment:
It is not entirely clear what this method is for. Why not just write if (user.getEmail()!=null) ?

Reply:
I deleted isValidInput() method and simply use  if (user.getEmail()!=null) ?. The reason why i did was i used this in different places